### DISCUSSION TITLE
Change default text wrapping to PANGO_WRAP_WORD_CHAR

### DISCUSSION THREAD

**Discussion by machineonamission** [libvips/libvips#3126] [Ideas]
Per [the Pango docs](https://docs.gtk.org/Pango/enum.WrapMode.html), WORD_CHAR wrapping will

> Wrap lines at word boundaries, but fall back to character boundaries if there is not enough space for a full word.

Currently, it uses WORD wrapping which means that if the width of a single word exceeds the `width` property, it will be clipped

![image](https://user-images.githubusercontent.com/35177464/198903351-b6767c34-355c-414b-aea3-8dc7b9565846.png)

Related to #2073, though I'm proposing a changed default which is probably easier to implement than a new API for selecting a mode.

**Comment by jcupitt** [comment]
I made a PR, let's discuss there. https://github.com/libvips/libvips/pull/3130
